cmd/execution_builder.go

```diff
@@ -1114,6 +1114,30 @@ func (exeNode *ExecutionNode) LoadGrpcServer(
 	), nil
 }
 
+func validateCheckpointRootHash(logger zerolog.Logger, bootstrapDir, filename string, expectedCommit flow.StateCommitment) error {
```

Comment:
We'll need this on ANs too. can this go somewhere the AN bootstrap could access?

Reply:
I refactored in CheckpointHasRoot so that AN can reuse it